lib/archethic_web/templates/explorer/chain.html.eex

```diff
@@ -81,14 +81,12 @@
                 <div class="level-item has-text-centered">
                     <div>
                         <p class="heading">UCO Balance</p>
-                        <p class="title"><%= @uco_balance %> UCO
+                        <p class="title"><%= to_float(@uco_balance) %> UCO
                         <%= if @uco_balance > 0 do %>
-                            (<%= format_usd_amount(@uco_balance, @uco_price[:usd]) %>)
+                          <%= format_usd_amount(@uco_balance, @uco_price[:usd]) %>)
```

Comment:
I think you inadvertently deleted the left parenthesis on this line, so the price isn't displayed between two parenthesis

Reply:
Done